node/tests/TestUtilities.ts

```diff
@@ -339,10 +341,44 @@ export function compareMaps(
     return JSON.stringify(map) == JSON.stringify(map2);
 }
 
+/**
+ * Check transaction response.
+ * @param response - Transaction result received from `exec` call.
+ * @param expectedResponseData - Expected result data from {@link transactionTest}.
+ */
+export function validateTransactionResponse(
+    response: ReturnType[] | null,
+    expectedResponseData: [string, ReturnType][],
+) {
+    const failedChecks: string[] = [];
+
+    for (let i = 0; i < expectedResponseData.length; i++) {
+        const [testName, expectedResponse] = expectedResponseData[i];
```

Comment:
```
    for (const expectedResponseDataValue in expectedResponseData) {
        const [testName, expectedResponse] = expectedResponseDataValue;
```

Reply:
I use `i` to get per-command response from `response`